client/src/containers/Courses/Courses.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
```

Comment:
These seem like big changes, as we're adding a new component. 

Is there a specific issue you're fixing? If yes, could we please reference it in the commit message and in the PR description?

Reply:
My new task is to make a front end Api for courses. I did it but I would like to show it in a table. I would like to get the data as an array not object to showing them in a table.